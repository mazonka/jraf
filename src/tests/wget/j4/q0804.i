command=jw md 0 /.jraf.sys/users : login a@b.cd * : read f82d3bb17612458c /home : put f82d3bb17612458c /home/204f03b4efdc5397/y 0 1 Kg== : read f82d3bb17612458c /home/204f03b4efdc5397/y