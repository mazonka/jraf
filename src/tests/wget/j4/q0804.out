OK /.jraf.sys/users : OK http://127.0.0.1:16000/jraf? : OK 0 -1 1 0 -1 204f03b4efdc5397 : OK 1 : OK 1 1 Kg==
==
204f03b4efdc5397
